SpatialGDK/Source/SpatialGDKFunctionalTests/SpatialGDK/UNR-3761/SpatialTestCrossServerRPC/NonReplicatedCrossServerRPCCube.h

```diff
@@ -0,0 +1,21 @@
+// Copyright (c) Improbable Worlds Ltd, All Rights Reserved
+
+#pragma once
+
+#include "CoreMinimal.h"
+#include "ReplicatedCrossServerRPCCube.h"
+#include "NonReplicatedCrossServerRPCCube.generated.h"
+
+UCLASS()
+class ANonReplicatedCrossServerRPCCube : public AReplicatedCrossServerRPCCube
```

Comment:
I think this is slightly confusing, and the base class here should just be ACrossServerRPCCube where the default state is replicated. The extended class can then modify that default state. Otherwise an instance of the derived class could still be referred to as a "replicated cube", when it's not.

Reply:
Yes, I can see how this might be confusing. I've put the base class back to its original name ACrossServerRPCCube .